internal/auth/password/repository_account.go

```diff
@@ -66,20 +67,20 @@ func (r *Repository) CreateAccount(ctx context.Context, scopeId string, a *Accou
 	var cred *Argon2Credential
 	if opts.withPassword {
 		if cc.MinPasswordLength > len(opts.password) {
-			return nil, fmt.Errorf("create: password account: password: %w", ErrTooShort)
+			return nil, errors.New(errors.PasswordTooShort, op, fmt.Sprintf("must be longer than %v", cc.MinPasswordLength))
 		}
 		if cred, err = newArgon2Credential(id, opts.password, cc.argon2()); err != nil {
-			return nil, fmt.Errorf("create: password account: %w", err)
+			return nil, errors.Wrap(err, op)
 		}
 	}
 
 	oplogWrapper, err := r.kms.GetWrapper(ctx, scopeId, kms.KeyPurposeOplog)
 	if err != nil {
-		return nil, fmt.Errorf("create: password account: unable to get oplog wrapper: %w", err)
+		return nil, errors.Wrap(err, op, errors.WithMsg("unable to get oplog wrapper"), errors.WithCode(errors.Encrypt))
```

Comment:
hum... is it really an encryption error or perhaps an invalid parameter?

Reply:
In other places in similar function calls I did encryption error

Reply:
Well I have gone through a search and we either do: `errors.Encrypt` or none and just leave it with a message :\

Reply:
no worries.. just a question.